test/token.test.js

```diff
@@ -332,3 +332,73 @@ describe('token.resetweekly', async assert => {
 
 })
 
+describe.only('transaction limits', async assert => {
```

Comment:
The test works fine, I just want to point out that the `only` function is doing that this test is the only one being executed

Reply:
Good catch thanks